src/nvim/highlight_defs.h

```diff
@@ -0,0 +1,126 @@
+#ifndef NVIM_HIGHLIGHT_DEFS_H
+#define NVIM_HIGHLIGHT_DEFS_H
```

Comment:
I moved them from globals to avoid an include loop. `syntax_defs.h` would be possible, but these are about ui highlights, not syntax.